src/main/java/screens/GraphScene.java

```diff
@@ -328,4 +330,60 @@ public INodeHandler getInfo() {
         return this.info;
     }
 
+    /**
+     * Determine the number of genome paths going through an edge.
+     * @param incNode the node edges will be drawn from.
+     * @param id the id of the node.
+     * @return An array with the number of paths going through each outgoing edge of the node.
+     */
+    private double[] determineEdgeWidth(Node incNode, int id) {
+        NodeGraph ng = NodeGraph.getCurrentInstance();
+        int[] outgoing = incNode.getOutgoingEdges();
+        double[] widths = new double[outgoing.length];
+        int maxInt = -1;
+        double maxX = -Double.MAX_VALUE;
+        for (int i = 0; i < outgoing.length; i++) {
+            DrawNode out = ng.getDrawNode(outgoing[i]);
+            if (out != null && out.getX() > maxX) {
+                maxX = out.getX();
+                maxInt = i;
+            }
+            widths[i] = getNumberOfDuplicates(id, outgoing[i]);
```

Comment:
Yeah, later it is used in a computation, which results in doubles, but here it can be still ints. Will be changed.